### ISSUE TITLE
Files from the library are saved in another folder

### ISSUE THREAD

**Issue by jonasraoni** [pkp/pkp-lib#9009]
**Describe the bug**
Context files are normally saved at `{$files_dir}/(journals|contexts|presses)/{$context_id}`, with the exception of the library files, which are always placed inside `{$files_dir}/contexts/`.

The fix is trivial, but will also require a migration to rename the folder.

**What application are you using?**
OJS 3.4rc2

**Comment by asmecher** [comment]
@jonasraoni, is this just a cosmetic issue with the folder naming? If so, I'd rather not ship this with 3.4.0-0 so late in the testing/release cycle.

**Comment by jonasraoni** [comment]
Yep, it's just something that I flagged, it has been this way since forever, so I don't see no reason to prioritize it. Probably will just make the life of people doing backups easier.
